vertical-pod-autoscaler/pkg/admission-controller/logic/pod_pre_processor.go

```diff
@@ -1,39 +0,0 @@
-/*
```

Comment:
What is the justification for moving `PodPreProcessor` from pod_pre_processor.go to pod_resource_handler.go?
Same question for `VpaPreProcessor`?

Reply:
I don't find it cleaner. We move a struct/interface from a file dedicated purely for it to other unrelated file while this struct/interface still remains public.
If it was private structure then I would consider it justified.

For now, please move back `PodPreProcessor` and `VpaPreProcessor` to their files. This PR is still huge without these additional changes.

Reply:
Done.